server/server.js

```diff
@@ -0,0 +1,92 @@
+const express = require("express");
+const cors = require("cors");
+const pool = require("./database");
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+
+app.post("/addUser", (req, res) => {
+  const firstName = req.body["firstName"];
+  const lastName = req.body["lastName"];
+  const email = req.body["email"];
+  const age = req.body["age"];
+  const userType = req.body["userType"];
+
+  console.log("First Name: " + firstName);
+  console.log("Last Name: " + lastName);
+  console.log("Email: " + email);
+  console.log("Age: " + age);
+  console.log("User Type: " + userType);
+
+  const insertSTMT = `INSERT INTO Users (FIRST_NAME, LAST_NAME, EMAIL, AGE, USER_TYPE, CREATED_AT, UPDATED_WHEN)
+    VALUES ('${firstName}', '${lastName}', '${email}', ${age}, '${userType}', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);`;
+
+  pool
+    .query(insertSTMT)
+    .then((response) => {
+      console.log("Data Saved");
+      console.log(response);
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+
+  console.log(req.body);
+  res.send("Response Received: " + JSON.stringify(req.body));
+});
+
+app.post("/addCourse", (req, res) => {
```

Comment:
Something that we could improve is utilizing HTTP vers to a resource.

Let's say our resource is courses and lessons.

To add stuff in courses and lessons. we could have a route:
- POST /courses
- POST /lessons

We're also using plural nouns for a resource URL. 
Here's a good read: https://restfulapi.net/resource-naming/ 😁

```suggestion
app.post("/courses", (req, res) => {
```

Reply:
Additional examples: ✌️

POST /courses - adds a course
PUT /courses/:courseId - updates the entire course record
PATCH /courses/:courseId - updates parts of the field of a course
DELETE /courses/:courseId - deletes a course